docs/root/source/transaction-concepts_ko.md

```diff
@@ -0,0 +1,60 @@
+<!---
+Copyright BigchainDB GmbH and BigchainDB contributors
+SPDX-License-Identifier: (Apache-2.0 AND CC-BY-4.0)
+Code is Apache-2.0 and docs are CC-BY-4.0
+--->
+
+# Transaction Concepts
```

Comment:
This heading is causing a problem with Sphinx:
```text
Warning, treated as error:
/home/travis/build/bigchaindb/bigchaindb/docs/root/source/transaction-concepts_ko.md:7:duplicate label transaction concepts, other instance in /home/travis/build/bigchaindb/bigchaindb/docs/root/source/transaction-concepts.md
```
You could probably fix it by adding some Korean text to this heading.

I suspect that if you do that, then the same issue will arise with the **TRANSFER Transactions** heading below, so you could add some Korean text to it as well.

Reply:
Yeah...T_T You so kind. Thank you so much. I'm sorry to bother you.